WalletWasabi.Fluent/Views/Wallets/Send/TransactionSummary.axaml

```diff
@@ -48,7 +48,7 @@
       </Button>
       <c:PreviewItem Icon="{StaticResource btc_logo}"
                      Label="Amount" HorizontalContentAlignment="Stretch"
-                     CopyableContent="{Binding Amount}">
+                     CopyableContent="{Binding CopyableAmount}">
```

Comment:
Just do `{Binding Amount.Btc}`

Reply:
Done in [0b1850f](https://github.com/zkSNACKs/WalletWasabi/pull/11787/commits/0b1850fbdbfebacf2d23ecc85d694102deb6398b)